touchforms/formplayer/static/formplayer/script/fullform-ui.templates.js

```diff
@@ -0,0 +1,48 @@
+this["JST"] = this["JST"] || {};
+
+this["JST"]["formplayer/templates/formplayer/fullform-ui/form.html"] = function(obj) {
+obj || (obj = {});
+var __t, __p = '', __e = _.escape;
+with (obj) {
+__p += '<div>\n    <h1 id="title"></h1>\n    <div id="form"></div>\n    <input id="submit" type="submit" value="Submit" />\n</div>\n';
```

Comment:
Ugh. Vellum when through a phase like that and it was painful. We even used grunt for a while, but that's another topic. The source code got out of sync with the build artifacts and confusion and buggy deploys ensued. What about making a [deploy script like Vellum has](https://github.com/dimagi/commcare-hq/blob/master/scripts/vellum-to-hq) to prepare a new version of touchforms for deploy? The vellum script is run manually to push a new version of vellum to HQ, and this does not happen on nearly every deploy, but it is the one true way to get a new version of Vellum into HQ.